spec/datetimeformat.html

```diff
@@ -1187,15 +1193,15 @@ <h1>ToLocalTime ( _t_, _calendar_, _timeZone_ )</h1>
           </thead>
           <tr>
             <td>[[Weekday]]</td>
-            <td>`WeekDay(tz)` specified in es2023's <emu-xref href="#sec-week-day">Week Day</emu-xref></td>
+            <td>WeekDay(𝔽(truncate(_tz_ / 10<sup>6</sup>)))</td>
```

Comment:
Good catch! This didn't occur to me before, but you're right that `floor` would be in line with how we otherwise round epoch-based exact times in Temporal. In fact this is currently inconsistent in Temporal's epochSeconds/epochMilliseconds/epochMicroseconds properties, so I'll open a follow-up issue over there.

Reply:
(It wouldn't make any observable difference until we actually have a source of epoch nanoseconds that isn't evenly divisible by 10⁶ to use as input to ToLocalTime, though.)